terraform/aptos-node/aws/kubernetes.tf

```diff
@@ -115,7 +115,7 @@ locals {
         value  = "validators"
         effect = "NoExecute"
       }]
-      remoteLogAddress = var.enable_logger ? "${helm_release.logger[0].name}-aptos-logger:5044" : null
+      remoteLogAddress = var.enable_logger ? "${helm_release.logger[0].name}-aptos-logger.${helm_release.logger[0].namespace}.svc:5044" : null
```

Comment:
can you explain to me how this works? are we running a log forwarder in the k8s namespace itself or something?

Reply:
We're running a log forwarder in the default namespace, since that's where we assume most services are installed for `aptos-node` (see https://github.com/aptos-labs/aptos-core/blob/main/terraform/aptos-node/aws/kubernetes.tf#L147). Of course the reliance on `default` namespace is not ideal, so this is why I'm doing `helm_release.logger[0].namespace` instead of ` default`, so we can make the namespace customizable in a later change.

For each cluster, there's only one installation of the log forwarder. In clusters like Forge with many `aptos-node`s installed across many namespaces, it's necessary to address the log forwarder with the full service name (including the namespace)

Reply:
I see. Yeah we should move to a k8s daemonset for the log collection and forwarding then this wouldn't be necessary anymore.